Revise overview for Install Tools tasks (#22675)
kubernetes/website: content/en/docs/tasks/tools/_index.md

title: "Install Tools"
description: Set up Kubernetes tools on your computer.
weight: 10
no_list: true
---

## kubectl

The Kubernetes command-line tool, `kubectl`, allows you to run commands against
Kubernetes clusters. You can use kubectl to deploy applications, inspect and manage
cluster resources, and view logs.

> [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/) explains how to
> download and install `kubectl` and set it up for accessing your cluster.

Reviewer: I like this sentence ✨ , but it's better practice to avoid starting sentences with links. Might we rephrase this? Feel free to reword to your satisfaction.
```suggestion
See [Install and Set Up kubectl](/docs/tasks/tools/install-kubectl/) for information about how to
download and install `kubectl` and set it up for accessing your cluster.
```

You can also read the [`kubectl` reference documentation](/docs/reference/kubectl/).

## Minikube

> [Minikube](https://minikube.sigs.k8s.io/) is a tool that makes it easy to run

Reviewer: Avoid "easy", "just", "simply", etc.

```suggestion
[Minikube](https://minikube.sigs.k8s.io/) is a tool that lets you run
```

Kubernetes locally. Minikube runs a single-node Kubernetes cluster on your personal
computer (including Windows, macOS and Linux PCs) so that you can try out Kubernetes,
or for daily development work.

You can follow the official [Get Started!](https://minikube.sigs.k8s.io/docs/start/)
guide, or read [Install Minikube](/docs/tasks/tools/install-minikube/) if your focus
is on getting the tool installed.

Once you have Minikube working, you can use it to
[run a sample application](/docs/tutorials/hello-minikube/).

## kind

> [kind](https://kind.sigs.k8s.io/docs/) is another tool for running Kubernetes locally.

Reviewer: Same here for starting a sentence with a link, to your satisfaction, etc. Perhaps:
```suggestion
kind is [another tool](https://kind.sigs.k8s.io/docs/) for running Kubernetes locally.
```

Unlike Minikube, which offers several different ways to run the cluster, kind requires
that you have [Docker](https://docs.docker.com/get-docker/) installed and configured.

> [Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start/) runs through what you

Reviewer: ```suggestion
[Quick Start](https://kind.sigs.k8s.io/docs/user/quick-start/) shows you what you
```
